### ISSUE TITLE
Add commentary info to workflow templates

### ISSUE THREAD

**Issue by mcdittmar** [ivoa-std/ivoatex#28]
I've been going through the process of setting up the TransformDM repo in GitHub.
I could see these workflow files in the other repository, but couldn't figure out where they came from.
It would help if the template files had some comment text at the top to indicate where they came from and maybe a pointer to the instructions for installing them in your repository.

eg: 
"# This file generated from a template file maintained in the ivoatex repository."
"# See \<link\> for instructions on how to create and install it into a project repository."

**Comment by msdemlei** [comment]
On Thu, Nov 19, 2020 at 09:19:31AM -0800, Mark Cresitello-Dittmar wrote:
> I could see these workflow files in the other repository, but couldn't figure out where they came from.
> It would help if the template files had some comment text at the top to indicate where they came from and maybe a pointer to the instructions for installing them in your repository.
> 
> eg: 
> # This file generated from a template file maintained in the ivoatex repository.
> # See <link> for instructions on how to create and install it into a project repository.

Sounds reasonable -- would you put it in?  For the URL, I'd suggest
to point to ivoatexdoc -- it doesn't say anything about it yet, but
it will have to in the end.

**Comment by mcdittmar** [comment]
You mean the [IVOA Note](https://ivoa.net/documents/Notes/IVOATex/index.html)?
Wouldn't it be better to point to the [README.rst](https://github.com/mcdittmar/ivoatex#automatic-pdf-preview-in-github) since it has the make instructions... at least until the document is updated and becomes the better resource?  I'd hate to send the next person down a dead-end street.

**Comment by mcdittmar** [comment]
I've made the changes in my fork, can submit a pull request, but have a question first..
The text looks rather odd when sitting IN the template..  maybe this should be added during the build process?
If so.. I won't bother with the PR.

**Comment by msdemlei** [comment]
On Thu, Nov 19, 2020 at 11:52:47AM -0800, Mark Cresitello-Dittmar wrote:
> I've made the changes in my fork, can submit a pull request, but
> have a question first..  The text looks rather odd when sitting IN
> the template..  maybe this should be added during the build
> process?  If so.. I won't bother with the PR.

You mean, because people might thing it refers to the template rather
than the installed thing?

You have a point there, but I'd say the extra tooling isn't worth the
benefit of making things a bit prettier -- people poking around in
the ivoatex folder ought to be prepared to see wires sticking out
occasionally.

On referencing README instead of ivoatexDoc... ok by me, though
historically I've done a far worse job keeping the README up to date
than ivoatexDoc.  On the other hand, as ivoatex maintenance becomes a
bit more collaborative, perhaps that'll improve...?

An alternative that wouldn't require any referencing that could
break:

```
# To enable github previews in ivoatex documents, do:
# make github-preview
# git commit
# git push
```

I'd expect people touching the machinery will quite certainly spot
(and, if necessary, update) it, it sits nicely in the template, and
it's probably a sufficient pointer for people seeing the workflows in
other peoples' repos.

**Comment by mcdittmar** [comment]
The alternative seems like a good starting point..
I've implemented that, with a little change to the phrasing. PR submitted
I'm still getting comfortable with the workflow, so hope its done right!

**Comment by msdemlei** [comment]
Fixed by PR #29.
